public/api/conf/1.0/application.yaml

```diff
@@ -3,20 +3,22 @@ info:
   title: Bulk Data File List
   version: "1.0"
 servers:
-  - url: https://test-api.service.hmrc.gov.uk/bulk-data-download
+  - url: https://test-api.service.hmrc.gov.uk
     description: Sandbox
-  - url: https://api.service.hmrc.gov.uk/bulk-data-download
+  - url: https://api.service.hmrc.gov.uk
```

Comment:
I think you need to keep the context in these urls. The API Platform's [guidance](https://confluence.tools.tax.service.gov.uk/display/ApiPlatform/OpenAPI+Specification+structure+and+style#OpenAPISpecificationstructureandstyle-ServersSection) says:
> You MUST copy your API context (from the definition.json file) into the OpenAPI specification by adding it the the servers section as shown below (for a made up context of test/madeup))